src/toxic.h

```diff
@@ -43,6 +43,7 @@
 #include <curses.h>
 
 #include <tox/tox.h>
+#include <tox/tox_private.h>
```

Comment:
This will make it so you can't release to Debian, because you're going to unconditionally rely on unstable APIs. Is that intentional?

Reply:
I'd make it an optional flag in the build. It's going to depend on experimental APIs that won't be stable across minor or patch releases, so it'll break in hilarious ways when Debian updates toxcore but not toxic or vice versa.

Reply:
or, less portable, dlopen()+dlsym() symbol searching (might not work if not exported) + needs to be conditional too.

Reply:
That won't know the type of the function, which may change.

Reply:
Good point. Added a makefile option that's disabled by default, but enabled in the static build.